composer/loggers/console_logger.py

```diff
@@ -79,6 +80,12 @@ def log_hyperparameters(self, hyperparameters: Dict[str, Any]):
         # Lazy logging of hyperparameters.
         self.hparams.update(hyperparameters)
 
+    def log_table(self, columns: List[str], rows: List[List[Any]], name: str = 'Table') -> None:
+        import pandas as pd
+        table = pd.DataFrame.from_records(data=rows, columns=columns).to_json(orient='split', index=False)
+        self.tables[name] = str(table)
+        del table
```

Comment:
is this needed? does it not destruct properly?

Reply:
good catch! not needed.